common/models/wordbreakers/src/main/default/data.ts

```diff
@@ -1,9 +1,13 @@
 // Automatically generated file. DO NOT MODIFY.
+// The generator script is defined at /common/models/wordbreakers/src/data-compiler/index.ts.
```

Comment:
Now that we are able to generate this from scripts in-repo, we could probably avoid checking this file in now (and perhaps change the extension to `.inc.ts`).

That said, the data hasn't been updated in a while, so keeping it here for now, at this stage of the reviewing process, highlights what updating the data will change.